tests/Tests/500__jupyterhub/test-jupyterlab-time.robot

```diff
@@ -0,0 +1,93 @@
+*** Settings ***
+Resource            ../../Resources/ODS.robot
+Resource            ../../Resources/Common.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource            ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Library             ../../../../libs/Helpers.py
+Library             DateTime
+Library             OpenShiftCLI
+Library             DebugLibrary
+
+Suite Setup         Load Spawner Page
+Suite Teardown      End Web Test
+
+
+*** Variables ***
+@{LIST_OF_IMAGES}       s2i-minimal-notebook    s2i-generic-data-science-notebook
+...                     pytorch                 tensorflow    minimal-gpu
+
+${LIMIT_TIME} =    40
+
+*** Test Cases ***
+Verify Average Spawn Time Is Less Than 40 Seconds
+    [Documentation]    Verifies that average spawn time for all JupyterHub images is less than 40 seconds
+    [Tags]    ODS-691
+    ...       Tier2
+    ${len} =    Get Length    ${LIST_OF_IMAGES}
+    ${avg_time} =    Get Average Time For Spawning    ${len}
+    Log    total_avg time to spawn ${avg_time}
+    Average Spawning Time Should Be Less Than    ${avg_time}    ${LIMIT_TIME}
+
+
+*** Keywords ***
+Load Spawner Page
+    [Documentation]    Suite Setup, loads JH Spawner
+    Begin Web Test
+    Launch JupyterHub Spawner From Dashboard
+
+Get Average Time For Spawning
+    [Documentation]    Returns the total of average time for spawning the images
+    [Arguments]    ${number_of_images}
+    ${total_time} =    Set Variable
+    FOR    ${image}    IN    @{LIST_OF_IMAGES}
+        ${avg} =    Set Variable
+        Spawn and Stop Server    ${image}
+        FOR    ${counter}    IN RANGE    4
+            Close Previous Tabs
+            ${sum} =    Spawn and Stop Server    ${image}
+            ${avg} =    Evaluate    ${avg} + ${sum}
+        END
+        ${avg} =    Evaluate    ${avg}/4
+        ${total_time} =    Evaluate    ${total_time} + ${avg}
+    END
+    ${average_time} =    Evaluate    ${total_time} / ${number_of_images}
+    [Return]    ${total_time}
+
+Average Spawning Time Should Be Less Than
+    [Documentation]    Checks than average time is less than ${time}
+    [Arguments]    ${avg_time}    ${time}
+    ${result} =    lt    ${avg_time}.0    ${time}.0.0
```

Comment:
You need to update this to do a simple integer/float comparison, e.g.
```
${result} =  Evaluate  float(${avg_time}) < float(${time})
```

Reply:
Done